src/Core/SettingsChangesHistory.cpp

```diff
@@ -628,8 +628,8 @@ const VersionToSettingsChangesMap & getMergeTreeSettingsChangesHistory()
         addSettingsChanges(merge_tree_settings_changes_history, "25.1",
         {
             {"enable_max_bytes_limit_for_min_age_to_force_merge", false, false, "Added new setting to limit max bytes for min_age_to_force_merge."},
-            {"add_minmax_index_for_numeric_columns", false, false, "New setting"},
-            {"add_minmax_index_for_string_columns", false, false, "New setting"},
+            {"add_minmax_index_for_numeric_columns", true, true, "New setting"},
```

Comment:
Shouldn't this be `false, true`? Old compatibility value should be off by default (unless we really want the opposite for some reason)